src/utils/link.utils.ts

```diff
@@ -0,0 +1,21 @@
+// This file holds all relevant link utils
+// TODO: bring all link utils from general.utils.ts to here
+
+export async function get(bankAccount: string): Promise<boolean> {
```

Comment:
comment(blocking): where is this used?

also, we should probably use better naming